linode/manager
# M3-5941: Linode landing page – Empty state

## Description 📝

Adds helpful links to the bottom of the Database and Linodes empty landing pages. These links also send GA events.

## Preview 📷

Databases Before:
![image](https://user-images.githubusercontent.com/2135749/193641093-0028fbbe-28ea-4498-b56f-626dc3ae29a1.png)
Databases After: 
![image](https://user-images.githubusercontent.com/2135749/193641629-16f673b6-bba8-4c97-ac43-b817747887a4.png)

Linodes Before:
![image](https://user-images.githubusercontent.com/2135749/193641768-0d275486-dc19-467f-8e9d-69ddc5854eb3.png)
Linodes After:
![image](https://user-images.githubusercontent.com/2135749/193641836-fb741a9d-eb2d-457f-90a5-34edfd9f1298.png)


## How to test 🧪

Ensure that the landing pages for Databases and Linodes are present and take the user to the correct location. In addition check the network tab to ensure that GA events are being sent.
